Automattic/jetpack
# Fix Lazy Load for Spotify Shortcode

- Implemented lazy loading and added test case
- updated changelog for spotify lazy loading
- [not verified] fix some phpcs errors





#### Changes proposed in this Pull Request:

* Adds lazy loading to spotify embeds, following the work of #20594.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No
#### Testing instructions:






* Regression test the spotify embed player, make sure it works with lazy load enabled.
* More testing instructions here: https://github.com/Automattic/jetpack/pull/20594
